ts/parts/Pointer.ts

```diff
@@ -623,18 +623,16 @@ class Pointer {
      * @private
      * @function Highcharts.Pointer#drop
      *
-     * @param {global.Event} e
-     *
-     * @return {void}
+     * @param {global.Event} evt
      */
-    public drop(e: Event): void {
+    public drop(evt: Event): void {
```

Comment:
Why was this renamed to `evt`? In all other aspects, we use `e` for event arguments, so we should keep consistent whenever we can.